conjure-java-core/src/main/java/com/palantir/conjure/java/types/BeanBuilderGenerator.java

```diff
@@ -316,18 +359,58 @@ private static List<TypeSpec> generateStageInterfaces(
         return interfaces.stream().map(TypeSpec.Builder::build).collect(Collectors.toList());
     }
 
+    private static TypeSpec generateNonStrictBuilderFinalStageInterface(
+            ClassName objectClass, TypeMapper typeMapper, List<EnrichedField> fields, SafetyEvaluator safetyEvaluator) {
+        ClassName completedStageClass = stageBuilderInterfaceName(objectClass, "completed_");
+        return TypeSpec.interfaceBuilder(completedStageClass)
+                .addModifiers(Modifier.PUBLIC)
+                .addMethod(MethodSpec.methodBuilder("build")
+                        .addModifiers(Modifier.PUBLIC, Modifier.ABSTRACT)
+                        .returns(Primitives.box(objectClass))
+                        .build())
+                .addMethods(fields.stream()
+                        .flatMap(field -> Streams.concat(
+                                Stream.of(generateMethodForFinalStageInterfaceField(
+                                        field, typeMapper, completedStageClass, safetyEvaluator)),
+                                generateAuxiliaryMethodsForFinalStageInterfaceField(
+                                        field, typeMapper, completedStageClass, safetyEvaluator)
+                                        .stream()))
```

Comment:
I want the [cons](https://en.wikipedia.org/wiki/Cons) operation in Java. The closest I found was `<E> Lists.asList(E first, E[] rest)`.